### PR TITLE
Added PhysicsFS

### PR DESCRIPTION
**Pull request by Mailaender**
https://github.com/search?q=org%3Aflathub+physfs&type=code and everyone is using the same version.

### CODE REVIEW

File: physfs/physfs.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "name": "PhysicsFS",
+    "buildsystem": "cmake-ninja",
+    "config-opts": [ "-DPHYSFS_BUILD_TEST=OFF" ],
+    "cleanup": [
+        "/lib/*.a",
```
Comment (Erick555) [inline physfs/physfs.json:6]: Would just `*.a` work?

File: physfs/physfs.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "/lib/pkgconfig",
+        "/bin"
+    ],
+    "sources": [
+        {
+            "type": "archive",
+            "url": "https://icculus.org/physfs/downloads/physfs-3.0.2.tar.bz2",
+            "sha256": "304df76206d633df5360e738b138c94e82ccf086e50ba84f456d3f8432f9f863",
+            "size": 194888,
+            "x-checker-data": {
```
Comment (hfiguiere) [inline physfs/physfs.json:17]: I don't think the x-checker-data works with shared-modules.

**Comment by hfiguiere** [comment]
> https://github.com/search?q=org%3Aflathub+physfs&type=code and everyone is using the same version.

and it's really trivial do build. Doesn't require special patches or hackery.

File: physfs/physfs.json (hunk 1 above)
Comment (hfiguiere) [reply physfs/physfs.json:6]: also the config `-DPHYSFS_BUILD_STATIC=OFF` would also do it (better)

**Comment by Mailaender** [comment]
Updated.

**Comment by TingPing** [comment]
Need somebody added to `CODEOWNERS` to commit to updating it.

**Comment by Mailaender** [comment]
Added myself.

**Comment by barthalion** [comment]
Please make sure to submit pull requests to repos which could use it.

**Pull request merged**